Backpack/CardButton/Classes/CardButtonViewConfigurator.swift

```diff
@@ -0,0 +1,50 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Hold shared logic between BPKSaveCardButton & BPKShareCardButton
+final class CardButtonViewConfigurator {
+
+    // CardButton Required a larger hit area of at least 48px
+    let widthHeight = BPKSpacingLg * 2
+
+    func createContainedBackgroundCircle() -> UIView {
```

Comment:
Shouldn't this view be part of the button?

Reply:
I will move this logic to a private parent BPKCardButton class 
This should move the logic to be as part of the button

Reply:
I move the logic to CardButtonProtocol
dcc8b9c3ec8d85fea61084f57e6fd95a39e99358